<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IP Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="mode_tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_client">
     <property name="whatsThis">
      <string/>
     </property>
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QGroupBox" name="packet_settings">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>471</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Packet Settings</string>
      </property>
      <widget class="QSlider" name="packet_size">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="packet_size_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Packet size:</string>
       </property>
      </widget>
      <widget class="QSlider" name="packet_num">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="packet_num_label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of packets:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="packet_size_val">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="packet_num_val">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="data_settings">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Source</string>
      </property>
      <widget class="QLineEdit" name="file_val">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="file_browse">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>51</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="file_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="data_settings_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Destination Settings</string>
      </property>
      <widget class="QLineEdit" name="file_val_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="file_label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Destination IP Address:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_server">
     <property name="toolTip">
      <string/>
     </property>
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QGroupBox" name="server_log">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>481</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Log</string>
      </property>
      <widget class="QTextBrowser" name="server_log_output">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>461</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="protocol_settings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Protocol Settings</string>
    </property>
    <widget class="QRadioButton" name="protocol_tcp_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="protocol_udp_radio">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="port_val">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>7000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="port_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
